algorithms/CPlusPlus/README.md

```diff
@@ -33,6 +33,8 @@
 4. [Finding squareroot using Binary search](Searching/sqrt-monotonic-binary-search.cpp)
 5. [KMP String Searching](Searching/kmp.cpp)
 6. [Ternary Search](Searching/Ternary-search.cpp)
+7. [Interpolation Search](Searching/interpolation-search.cpp)

```

Comment:
You have added 2 files here but I can only view the Exponential Search file. The interpolation Search file is missing.

Reply:
The interpolation search is already there in the repo, the contributor must have forgot to add it the readme

Reply:
Okayy! That's great!